src/main/g8/build.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'application'
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    compile 'com.typesafe.akka:akka-http_2.12:$akka_http_version$'
+    compile 'com.typesafe.akka:akka-stream_2.12:$akka_version$'
+    testCompile 'com.typesafe.akka:akka-http-testkit_2.12:$akka_http_version$'
+    testCompile 'org.scalatest:scalatest_2.12:3.0.1'
```

Comment:
Without scalatest I get this error:

```
[info] [info] Compiling 3 Java sources to /tmp/sbt_823bc44b/scripted/target/scala-2.12/test-classes...
[info] [error] /tmp/sbt_823bc44b/scripted/src/test/java/com/example/HttpServerHttpAppTest.java:10: cannot access org.scalatest.junit.JUnitSuiteLike
[info] [error]   class file for org.scalatest.junit.JUnitSuiteLike not found
[info] [error] public class HttpServerHttpAppTest extends JUnitRouteTest {
[info] [error] /tmp/sbt_823bc44b/scripted/src/test/java/com/example/HttpServerHttpAppTest.java:17: cannot find symbol
[info] [error]   symbol:   method testRoute(akka.http.javadsl.server.Route)
[info] [error]   location: class com.example.HttpServerHttpAppTest
[info] [error]     appRoute = testRoute(server.route());
[info] [error] (test:compileIncremental) javac returned nonzero exit code
```

Maybe it is worth creating a ticket on akka/akka-http about this, WDYT?